tests/test_fragments.py

```diff
@@ -15,24 +15,16 @@
         ("hello world!!%^(@after)writing a lot\n", 1),
         ("hello world!!%^(@after)writing a lot\n\n", 1),
         ("", 0),
-        ("", 0),
-        (" ", 0),
```

Comment:
@gkapfham these tests are original code from before we started on this branch. Is there any use to duplicate test cases? Perhaps to catch flaky tests?

Reply:
Hi @Michionlion, I don't think that those two duplicate tests for `""` are both needed.